vj4/ui/templates/record_main_tr.html

```diff
@@ -21,7 +21,7 @@
     *
   {% endif %}
   </td>
-  <td class="col--submit-by"><a href="{{ reverse_url('user_detail', uid=rdoc['uid']) }}">{{ udoc['uname'] }}</a></td>
+  <td class="col--submit-by"><a href="{{ reverse_url('user_detail', uid=rdoc['uid']) }}">{{ dudoc['name_alias'] if dudoc['name_alias'] else udoc['uname'] }}</a></td>
```

Comment:
`dudoc['name_alias']|default(udoc['uname'])`

Please note this only works when `dudoc['name_alias']` evaluates to `undefined` and that should be preferred.

Reply:
This is already fixed yesterday. It seems you are not reviewing the latest code.